airflow/migrations/versions/6e96a59344a4_make_taskinstance_pool_not_nullable.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Make TaskInstance.pool not nullable
+
+Revision ID: 6e96a59344a4
+Revises: 939bb1e647c8
+Create Date: 2019-06-13 21:51:32.878437
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+from airflow.models.pool import Pool
+from airflow.models.taskinstance import TaskInstance
```

Comment:
Could we do something like https://github.com/apache/airflow/blob/8644a6f94c345b10dce46b9c71a02c2c1285bc85/airflow/migrations/versions/cc1e65623dc7_add_max_tries_column_to_task_instance.py#L46-L53 instead?

(We've had problems in the past with migrations where the model changing in "the future" broken an older migration making upgrades harder)

Reply:
That make sense. Fixed.